modules/telco-core-monitoring.adoc

```diff
@@ -10,16 +10,21 @@ New in this release::
 * No reference design updates in this release
 
 Description::
-The {cmo-first} is included by default in {product-title} and provides monitoring (metrics, dashboards, and alerting) for the platform components and optionally user projects as well.
-+
-[NOTE]
-====
-The default handling of pod CPU and memory metrics is based on upstream Kubernetes `cAdvisor` and makes a tradeoff that prefers handling of stale data over metric accuracy. This leads to spiky data that will create false triggers of alerts over user-specified thresholds. OpenShift supports an opt-in dedicated service monitor feature creating an additional set of pod CPU and memory metrics that do not suffer from the spiky behavior.
-For additional information, see link:https://access.redhat.com/solutions/7012719[Dedicated Service Monitors - Questions and Answers].
-====
+The Cluster Monitoring Operator (CMO) is included by default in {product-title} and provides monitoring (metrics, dashboards, and alerting) for the platform components and optionally user projects.
+You can customize the default log retention period, custom alert rules, and so on.
+The default handling of pod CPU and memory metrics, based on upstream Kubernetes and cAdvisor, makes a tradeoff favoring stale data over metric accuracy.
+This leads to spikes in reporting, which can create false alerts, depending on the user-specified thresholds.
+OpenShift supports an opt-in Dedicated Service Monitor feature that creates an additional set of pod CPU and memory metrics that do not suffer from this behavior.
+For more information, see link:https://access.redhat.com/solutions/7012719[Dedicated Service Monitors - Questions and Answers (Red Hat Knowledgebase)].
```

Comment:
```suggestion
For more information, see the link:https://access.redhat.com/solutions/7012719[Dedicated Service Monitors - Questions and Answers (Red Hat Knowledgebase)].
```

Reply:
Keeping this as is to conform with our style guide.